blocks/results-list-block/features/results-list/results/index.jsx

```diff
@@ -185,6 +205,14 @@ const Results = ({
 
   return (viewableElements?.length > 0 && !isServerSideLazy) ? (
     <div className="results-list-container">
+      {showOverline
```

Comment:
This is not required, as an overline is a story item.

Reply:
removed